docs/docs/features/bulk-upload.md

```diff
@@ -46,19 +46,31 @@ The API key can be obtained in the user setting panel on the web interface.
 
 ### Run via Docker
 
-Be aware that as this runs inside a container it mounts your current directory as a volume, and for the -d flag you need to use the path inside the container.
+:::caution Running inside Docker
+Be aware that as this runs inside a container, you need to mount the folder from which you want to import into 
+the container. The below command uses `$(pwd)` to mount the folder you are currently in.
+:::
+
+You can run the CLI inside of a docker container like this:
 
 ```bash
-docker run -it --rm -v $(pwd):/import ghcr.io/immich-app/immich-cli:latest upload --key HFEJ38DNSDUEG --server http://192.168.1.216:2283/api -d /import
```

Comment:
I think `--d` is a required flag?

Reply:
Check the linked CLI PR - I set a default via an env var in the Dockerfile.